python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -579,20 +573,25 @@ def explain_node(
 
         self.set_masks(batched_graph, edge_mask)
 
+        batched_feats = batched_graph.ndata["feat"]
         # the model prediction with the updated edge mask
         logits = self.model(
             batched_graph, batched_feats, edge_weight=self.edge_mask, **kwargs
         )
         probs = F.softmax(logits, dim=-1)
 
+        batched_inverse_indices = (
+            batched_graph.ndata["train"].nonzero().squeeze(1)
+        )
+
         if training:
             self.batched_feats = batched_feats
             probs = probs.data
         else:
             self.clear_masks()
 
         return (
-            probs.data,
+            probs,
```

Comment:
I think so far the code changes in `explain_node` are equivalent.  Any specific reason to write the code in your way instead of the original?

Reply:
mainly keep it uniform and hopefully more optimized. I didn't find myself needing any complicated book-keeping logic to implement this in `homogenous` form, but since its `hetero` implementation called for a bit more i figured it was best to also update the `homogenous` code to lean on dgl-optimized transformations

Reply:
To @BarclayII , I think the reason is to keep consistent with explain_graph.

Reply:
Sure.